internal/operators/common/common_test.go

```diff
@@ -0,0 +1,61 @@
+package common_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/operators/common"
+	"github.com/openshift/assisted-service/internal/operators/mce"
+	"github.com/openshift/assisted-service/internal/operators/odf"
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/conversions"
+)
+
+var _ = DescribeTable(
+	"Get valid disk count",
+	func(disks []*models.Disk, diskID string, minSize, expectedEligibleDisks, expectedAvailableDisks int64) {
+		eligibleDisks, availableDisks := common.GetValidDiskCount(disks, diskID, minSize)
+		Expect(eligibleDisks).To(Equal(expectedEligibleDisks))
+		Expect(availableDisks).To(Equal(expectedAvailableDisks))
+	},
+	Entry("no disk provided", []*models.Disk{}, "", int64(0), int64(0), int64(0)),
+	Entry("no valid disk provided", []*models.Disk{
+		{SizeBytes: 20 * conversions.GB, DriveType: models.DriveTypeUnknown, ID: "/dev/disk/by-id/disk-1"},
+		{SizeBytes: 20 * conversions.GB, DriveType: models.DriveTypeVirtual, ID: "/dev/disk/by-id/disk-2"},
+	}, "", int64(0), int64(0), int64(0)),
```

Comment:
Why are these all `int64`? How many disks are you expecting?

Reply:
You're right, I've just blindly kept the type that was already there but I guess we'll never have more than `int`

Reply:
Actually it's a bit of a mess, swagger sets default type for models to int64, we could change it to int within business logic and then convert it back to int64 when we're dealing with the generated models but it would be a bigger change - happy to do it even if it'd be a bit out of scope - WDYT?

Reply:
Ah alright if it's coming from swagger it's fine.